### ISSUE TITLE
Type mismatch between return type and phpdoc

### ISSUE THREAD

**Issue by rentalhost** [kalessil/phpinspectionsea#478]
It should check if we have some kind of mismatch of return type and phpdoc.

For instance:

```php
/**
 * @return string|null
 */
public function get(): string {
    return 'hello';
}
```

Return type should be `: ?string` or phpdoc should be `string` (we can provide two QFs for cases like that, and two inspection points).

**Comment by kalessil** [comment]
We are getting back to PhpDoc correctness topic. 
It's not a reliable source of information and I don't see enough benefits for the project to grow in this direction, hence declining this FR.

**Comment by rentalhost** [comment]
Oh, you are right. I will disable the `@return` inspection from PS, once that return type is more reliable. Thanks!

**Comment by kalessil** [comment]
No problems =)
